addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,222 @@ msgctxt "#802"
+msgctxt "#823"
+msgid "Repeating (guide-based)"
+msgstr ""
+
+#empty string with id 824
+
+#. Representation of PVR timer state "disabled"
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#825"
+msgid "Disabled"
+msgstr ""
+
+#. Text for notification that a repeating timer now has state "scheduled" (means enabled)
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#826"
+msgid "Repeating timer enabled"
+msgstr ""
+
+#. Notification text for PVR timer state "disabled"
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#827"
+msgid "Timer disabled"
+msgstr ""
+
+#. Text for notification that a repeating timer has been deleted
+#: xbmc/pvr/timers/PVRTimers.cpp
+msgctxt "#828"
+msgid "Repeating timer deleted"
+msgstr ""
+
+#empty string with id 829
+
+#. label for PVR Timer Weekdays settings dialog
+#: skin.confluence
+msgctxt "#830"
+msgid "Weekdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#831"
+msgid "Mondays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#832"
+msgid "Tuesdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#833"
+msgid "Wednesdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#834"
+msgid "Thursdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#835"
+msgid "Fridays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#836"
+msgid "Saturdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#837"
+msgid "Sundays"
+msgstr ""
+
+#empty strings from id 838 to 839
+
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+msgctxt "#840"
+msgid "You are about to delete a repeating timer. Do you also want to delete all timers currently scheduled by this timer?"
+msgstr ""
+
+#. Label of a context menu entry to create a new recording timer with advanced configuration options
+#: xbmc/pvr/windows/GUIWindowPVRGuide.cpp
+#: xbmc/pvr/windows/GUIWindowPVRSearch.cpp
+msgctxt "#841"
+msgid "Add advanced timer"
+msgstr ""
+
+#. Label of the option to switch between hierarchical and flat timer view (in confluence sideblade).
+#: skin.confluence
+msgctxt "#842"
+msgid "Group timers"
+msgstr ""
+
+#. Label of a context menu entry to enable a currently disabled timer
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
+msgctxt "#843"
+msgid "Enable timer"
```

Comment:
as this is shown in the context menu of a timer it could probably be reduced to just "enable" (and "disable"). In this case we could reuse the labels from the add-ons (24022 and 24021), or do you think they might require different translations here?
In case we reuse it would be nice to add according usage and contextual notes to the two existing labels.